.nvmrc

```
@@ -1 +1 @@
-lts/gallium
+18.12.0
```

let's go with:
```suggestion
lts/hydrogen
```
so we don't need to update patch here (unless renovate would do that for us)

---

then we need to remember to update to v20 when the time comes. Do you think that's ok?

---

nevermind, nvm supports `lts/*` which automatically uses the latest one 👍